data/elasticsearch/elasticsearch.go

```diff
@@ -86,3 +83,8 @@ func (es *ElasticSearch) All(esIndex string) []*elastic.SearchHit {
 	}
 	return searchResult.Hits.Hits
 }
+
+// GetNative returns the native elastic search client
+func (es *ElasticSearch) GetNative() *elastic.Client {
```

Comment:
Will any other consumers actually need access to the internal client? Subjective, but if so, consider renaming to `GetClient`.